Extensions/Leaderboards/JsExtension.js

```diff
@@ -0,0 +1,108 @@
+// @flow
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Changes in this file are watched and automatically imported if the editor
+ * is running. You can also manually run `node import-GDJS-Runtime.js` (in newIDE/app/scripts).
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+
+/*::
+// Import types to allow Flow to do static type checking on this file.
+// Extensions declaration are typed using Flow (like the editor), but the files
+// for the game engine are checked with TypeScript annotations.
+import { type ObjectsRenderingService, type ObjectsEditorService } from '../JsExtensionTypes.flow.js'
+*/
+
+module.exports = {
+  createExtension: function (
+    _ /*: (string) => string */,
+    gd /*: libGDevelop */
+  ) {
+    const extension = new gd.PlatformExtension();
+    extension
+      .setExtensionInformation(
+        'Leaderboards',
+        _('Leaderboards'),
+        _('Allow your game to send scores to your leaderboards.'),
+        'Florian Rival',
+        'Open source (MIT License)'
+      )
+      .setCategory('Leaderboards');
+    extension
+      .addInstructionOrExpressionGroupMetadata(_('Leaderboards'))
+      .setIcon('JsPlatform/Extensions/leaderboard.svg');
+
+    extension
+      .addAction(
+        'SavePlayerScore',
+        _('Save player score'),
+        _("Save the player's score to a giver leaderboard."),
+        _(
+          'Send to leaderboard _PARAM0_ the score _PARAM1_ with player name: _PARAM2_.'
+        ),
+        '',
+        'JsPlatform/Extensions/leaderboard.svg',
+        'JsPlatform/Extensions/leaderboard.svg'
+      )
+      .addParameter('leaderboardId', 'Leaderboard', '', false)
```

Comment:
Almost sure you'll need to pass the current scene too (search for "addCodeOnlyParameter" and "currentScene")

Reply:
Is it to get the playerUuid or did I miss something ?

Reply:
Yes it's to get the `runtimeScene` passed to your function (from which you can use `getGame` (https://docs.gdevelop.io/GDJS%20Runtime%20Documentation/classes/gdjs.RuntimeScene.html#getGame) to get the gdjs.RuntimeGame